lib/web_ui/lib/src/engine/platform_dispatcher.dart

```diff
@@ -455,8 +455,7 @@ class EnginePlatformDispatcher extends ui.PlatformDispatcher {
             contentManager: platformViewManager,
             contentHandler: (html.Element content) {
               domRenderer.glassPaneElement!.append(content);
-            }
-          );
+            });
```

Comment:
nit: I think I'd prefer a trailing comma in this case.

Reply:
done